certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3325 @@
+                      "type": "integer"
+                    },
+                    "suspicious": {
+                      "type": "integer"
+                    },
+                    "timeout": {
+                      "type": "integer"
+                    },
+                    "undetected": {
+                      "type": "integer"
+                    }
+                  },
+                  "required": [
+                    "harmless",
+                    "malicious",
+                    "suspicious",
+                    "timeout",
+                    "undetected"
+                  ]
+                },
+                "status": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "date",
+                "results",
+                "stats",
+                "status"
+              ]
+            },
+            "id": {
+              "type": "string"
+            },
+            "type": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "attributes",
+            "id",
+            "type"
+          ]
+        },
+        "meta": {
+          "type": "object",
+          "properties": {
+            "url_info": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string"
+                },
+                "url": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "id",
+                "url"
+              ]
+            }
+          },
+          "required": [
+            "url_info"
+          ]
+        }
+      },
+      "required": [
+        "data",
+        "meta"
+      ]
+    },
+    "fileReport": {
+      "type": "object",
+      "properties": {
+        "FileType": {
+          "type": "string"
+        },
+        "FileTypeExtension": {
+          "type": "string"
+        },
+        "LineCount": {
+          "type": "integer"
+        },
+        "MIMEEncoding": {
+          "type": "string"
+        },
+        "MIMEType": {
+          "type": "string"
+        },
+        "Newlines": {
+          "type": "string"
+        },
+        "WordCount": {
+          "type": "integer"
+        }
+      }
+    },
+    "urlReport": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string"
+            },
+            "type": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "filesReport": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "object",
+          "properties": {
+            "id": {
+              "type": "string"
+            },
+            "type": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "ipResult": {
```

Comment:
This is huge... Do you need all properties or can it be resricted to few important/most used properties? If yes, how are you expecting uses to use it? This looks like a usability issue

Reply:
Yes, we have review the requirement with the PM, and discuss the large amount of properties, After double verify it have come to understanding that it must have all the properties